Author: gregor herrmann <gregoa@debian.org>
Description: skip Perl::Critic tests, they tend to break with new Perl::Critic releases
Debian-specific: yes

--- a/t/perlcritic.t
+++ b/t/perlcritic.t
@@ -4,6 +4,8 @@
 use Test::More;
 use English qw(-no_match_vars);
 
+plan skip_all => "Skip Perl::Critic tests for Debian build.";
+
 unless (-e 't/author.enabled') {
     plan skip_all => "Author-only tests";
     exit 0;
